src/lammpsio/snapshot.py

```diff
@@ -253,13 +231,7 @@ def to_hoomd_gsd(self, type_map=None):
         frame.particles.N = self.N
         if self.has_position():
             # Center the positions using HOOMD tilt factors (computed above)
-            matrix = numpy.array(
-                [
-                    [L[0], tilt[0] * L[1], tilt[1] * L[2]],
-                    [0, L[1], tilt[2] * L[2]],
-                    [0, 0, L[2]],
-                ]
-            )
+            matrix = Box.to_matrix(self.box)[0]
             center = self.box.low + 0.5 * numpy.sum(matrix, axis=1)
```

Comment:
This would be how to do this if low is returned first:
```suggestion
            low, matrix = self.box.to_matrix()
            center = low + 0.5 * numpy.sum(matrix, axis=1)
```